File: hackerrank/day16/main.js
'use strict';

const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('', (S) => {

    try {
        const numero = Number(S);

        if (Number.isNaN(numero)) {
            throw new Error();
        }

        console.log(numero);
    }
    catch (error) {
        console.log('Bad String');
    }

    rl.close();
});

File: hackerrank/day16/main.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string S = Console.ReadLine();

        try
        {
            int numero = int.Parse(S);
            Console.WriteLine(numero);
        }
        catch (FormatException)
        {
            Console.WriteLine("Bad String");
        }
    }
}